### PR TITLE
test: new formatting and screenshots addition

### PR DESCRIPTION
**Pull request by olehbairak**
# What ❔

## Checklist

- [ ] PR title corresponds to the body of PR (we generate changelog entries from PRs).
- [ ] Tests for the changes have been added / updated.
- [ ] Documentation comments have been added / updated.

### CODE REVIEW

File: packages/testing-docs/docs/BE Tests/ Common /Networks navigation - Production/id565 - Networks navigation Prod - Common - Navigation to Sepolia.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+tags: ['Full test', 'manual', 'Navigation', 'Network', 'regression', 'To Automate', 'ZKF-2507', 'Active']
```
Comment (pcheremu) [inline packages/testing-docs/docs/BE Tests/ Common /Networks navigation - Production/id565 - Networks navigation Prod - Common - Navigation to Sepolia.md:2]: @olehbairak Do we decide to remove all ZKF-**** tags?

File: packages/testing-docs/docs/BE Tests/ Common /Networks navigation - Staging/id562 - Networks navigation Staging - Common - Navigation to zkSync Era Testnet.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+tags: ['Full test', 'manual', 'Navigation', 'Network', 'Smoke test', 'ZKF-2507', 'Automated']
```
Comment (pcheremu) [inline packages/testing-docs/docs/BE Tests/ Common /Networks navigation - Staging/id562 - Networks navigation Staging - Common - Navigation to zkSync Era Testnet.md:2]: Finally, can you please sort/select/take attention is the test case 'manual' or 'automated'? We have to keep only one of this tags.

File: packages/testing-docs/docs/BE Tests/Common /Blockscout API /id546 - API - Common - Get balance.md [hunk 3]
```diff
@@ -5,33 +5,43 @@ tags: ['API', 'Common', 'Full test', 'regression', 'ZKF-2450', 'Active']
...
+Example: https://zksync2-testnet.zkscan.io/api?module=account&action=tokenlist&address=0x6cC8cf7f6b488C58AA909B77E6e65c631c204784
 
+## Precondition
 
 ## Scenario
 - Prepare a custom token on L2
 - Make a request to show the balance
     - ?module=account&action=tokenlist&address=\{hash\}
 - You can see the token with its balance
+- ![Screenshot](../../../../static/img/screenshots/common/BlockscoutAPI/id546_1.png);
```
Comment (pcheremu) [inline packages/testing-docs/docs/BE Tests/Common /Blockscout API /id546 - API - Common - Get balance.md:41]: Looks good!

**Comment by pcheremu** [comment]
@olehbairak Please create a task to **clean extra ZKF tags and check if test case contains only one (manual or automated tag)**. feel free to merge this PR to the initial and create another one based on linear task or make this fix in the same PR. @amelnytskyi fyi

File: packages/testing-docs/docs/BE Tests/ Common /Networks navigation - Production/id565 - Networks navigation Prod - Common - Navigation to Sepolia.md (hunk 1 above)
Comment (olehbairak) [reply packages/testing-docs/docs/BE Tests/ Common /Networks navigation - Production/id565 - Networks navigation Prod - Common - Navigation to Sepolia.md]: Yes, fixed.
Thank you!

File: packages/testing-docs/docs/BE Tests/ Common /Networks navigation - Staging/id562 - Networks navigation Staging - Common - Navigation to zkSync Era Testnet.md (hunk 2 above)
Comment (olehbairak) [reply packages/testing-docs/docs/BE Tests/ Common /Networks navigation - Staging/id562 - Networks navigation Staging - Common - Navigation to zkSync Era Testnet.md]: Yes, fixed.
Thank you!

File: packages/testing-docs/docs/BE Tests/Common /Blockscout API /id546 - API - Common - Get balance.md (hunk 3 above)
Comment (olehbairak) [reply packages/testing-docs/docs/BE Tests/Common /Blockscout API /id546 - API - Common - Get balance.md:41]: Thank you!

**Comment by olehbairak** [comment]
> Later lets decide, do we want to have Automated and partly manual together. F.e. test id212 tags: ['Artifacts', 'Automated', 'Full test', 'Single-file', 'Smart Contract Verification page', 'Smoke test', 'Solidity', 'Active Partly Manual']

Yes, totally agree. We need to define whether these is the same, cause in fact they are partly automated, or to have only one identifier.

**Comment by amelnytskyi** [comment]
> Later lets decide, do we want to have Automated and partly manual together. F.e. test id212 tags: ['Artifacts', 'Automated', 'Full test', 'Single-file', 'Smart Contract Verification page', 'Smoke test', 'Solidity', 'Active Partly Manual']

Let's have only 'Active Partly Manual' for such cases - WDYT?

**Comment by pcheremu** [comment]
Yep, Lets keep only one tag the will describe the status. And be careful with signing that we unexpectedly see here.

**Review by pcheremu — CHANGES_REQUESTED** [review]
All commits should be signed

**Pull request closed without merging**